docs/embeddable-uis/element/access-request.mdx

```diff
@@ -0,0 +1,421 @@
+---
+sidebar_position: 3
+title: Access Request
+---
+
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Access Request Element
+
+<span className="subtitle-custom">
+The Access Request feature allows users to request access to restricted resources. The request is sent to the admin, who can approve or deny it.
+</span>
+<br/>
+<br/>
+
+<span style={{margin:'0 30px'}}>
+<img src="/img/approval-flows/ar-element.png" alt="ar-element" style={{width:'70%',border:'1px solid black'}}/>
+</span>
+
+
+## Overview
+
+Access Request gives us news capability in our app. It's easy to allow or deny access requests that our users request.
```

Comment:
@danielbass37  have you reviewed the English here?

Access Request gives us new capability in our app. Now the users can ask to dynamically change their own permissions if they want to access a forbidden action.